src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -123,13 +155,7 @@ public static function formatMemory(int $memory)
 
     public static function strlenWithoutDecoration(OutputFormatterInterface $formatter, ?string $string)
     {
-        $string = self::removeDecoration($formatter, $string);
-
-        if (preg_match('//u', $string)) {
-            return (new UnicodeString($string))->width(true);
```

Comment:
The `true` here is for "remove ansi decoration", but that is already done in `self::removeDecoration`.